src/client.rs

```diff
@@ -51,8 +51,8 @@ impl InstantQueryBuilder {
     }
 
     /// Include an additional parameter to the request.
-    pub fn param(mut self, name: &'static str, value: impl Into<String>) -> Self {
-        self.params.push((name, value.into()));
+    pub fn param(mut self, name: &'static str, value: impl ToString) -> Self {
```

Comment:
Let's rename this to `query` in accordance with `reqwest::RequestBuilder`. Then we are good to go!